packages/discord.js/src/structures/ApplicationCommand.js

```diff
@@ -396,6 +397,19 @@ class ApplicationCommand extends Base {
     }
     return true;
   }
+  
+  /**
+   * When concatenated with a string, this automatically returns the command's mention instead of the ApplicationCommand object.
```

Comment:
This should mention that it will only return the mention if the command is a chat input command

Reply:
Done!